Forms/MainForm.cs

```diff
@@ -1043,11 +1187,35 @@ private bool ValidateInputFile()
             return true;
         }
 
+        private bool ValidatePresetFile(String[] lines)
+        {
+            if (lines.Length > 0 && (lines[0].Equals("MMR Preset File [" + AssemblyVersion + "]") || lines[0].Equals("MMR Preset File [dev]")))
```

Comment:
Does this mean that with each new release, all previous config files will become invalid?